src/Workspaces/Core/MSBuild/Microsoft.CodeAnalysis.Workspaces.MSBuild.csproj

```diff
@@ -58,22 +72,15 @@
     <EmbeddedResource Update="WorkspaceMSBuildResources.resx" GenerateSource="true" />
   </ItemGroup>
   <!--
-    Include the BuildHost binaries into this package as well as a regular library. At the moment, we're including it in lib because we're internally using it as a library in-process for a few of our 
-    RPC interfaces with our remote process.
+    Include certain project reference binaries into this package in lib/ regular library.
     -->
   <PropertyGroup>
-    <TargetsForTfmSpecificBuildOutput>$(TargetsForTfmSpecificBuildOutput);GetBuildHostBinaries</TargetsForTfmSpecificBuildOutput>
-    <TargetsForTfmSpecificDebugSymbolsInPackage>$(TargetsForTfmSpecificBuildOutput);GetBuildHostDebugSymbols</TargetsForTfmSpecificDebugSymbolsInPackage>
+    <TargetsForTfmSpecificBuildOutput>$(TargetsForTfmSpecificBuildOutput);GetAdditionalBinariesToPackage</TargetsForTfmSpecificBuildOutput>
   </PropertyGroup>
-  <Target Name="GetBuildHostBinaries" DependsOnTargets="ResolveReferences">
-      <MSBuild Projects="..\MSBuild.BuildHost\Microsoft.CodeAnalysis.Workspaces.MSBuild.BuildHost.csproj" Targets="_GetBuildOutputFilesWithTfm">
-        <Output TaskParameter="TargetOutputs" ItemName="BuildOutputInPackage" />
-      </MSBuild>
-  </Target>
-  <Target Name="GetBuildHostDebugSymbols">
```

Comment:
I can't figure out what scenario symbol packing was happening in, so just removing it.